core/src/main/scala/com/raphtory/spouts/ResourceOrFileSpout.scala

```diff
@@ -0,0 +1,27 @@
+package com.raphtory.spouts
+
+import com.raphtory.api.input.{Spout, SpoutInstance}
+
+import scala.io.Source
+import scala.util.Try
+
+case class ResourceOrFileSpout(resource: String) extends Spout[String] {
```

Comment:
does this need docs?

Reply:
not very easy to do since it's running on the server side and if I move it into testing it doesn't exist on the server side